scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -63,7 +55,7 @@
                         "type": "Microsoft.Insights/dataCollectionRules",
                         "apiVersion": "2021-04-01",
                         "name": "[variables('dcrName')]",
-                        "location": "[parameters('workspaceLocation')]",
+                        "location": "[parameters('clusterRegion')]",
```

Comment:
There is  no "clusterRegion" parameter defined in the template so this will cause deployment failure.

Reply:
removed